server/mux.go

```diff
@@ -180,8 +181,19 @@ func NewMux(opts MuxOpts, service Service) http.Handler {
 	router.PUT("/chronograf/v1/sources/:id/dbs/:dbid/rps/:rpid", service.UpdateRetentionPolicy)
 	router.DELETE("/chronograf/v1/sources/:id/dbs/:dbid/rps/:rpid", service.DropRetentionPolicy)
 
+	// Set statusFeedURL to a default InfluxData JSON feed if not explicitly set by user
+	statusFeedURL := opts.StatusFeedURL
+	if statusFeedURL == "" {
+		statusFeedURL = "http://news.influxdata.com/feed.json"
+	}
+
+	externalLinks := getExternalLinksResponse{
```

Comment:
Rather than have mux.go be smart about what responses AllRoutes makes, it should just pass the URLs perhaps just the URL is given to AllRoutes. 

AllRoutes would then encode the URLs into whatever response object is needed. 

So, something like this:

```go
allRoutes := &AllRoutes{
	Logger:        opts.Logger,
	StatusURL:   statusFeedURL,
}
```

So, AllRoutes is in charge of marshaling responses, and mux.go is in charge of creating route handlers.